src/components/CreateEventForm/CreateEventForm.tsx

```diff
@@ -158,28 +165,32 @@ export default function CreateEventForm(
                     </FloatingLabel>
                 </Col>
             </Row>
-
             <Row>
                 <Col>
-                    <FloatingLabel
-                        controlId="eventStartTime"
-                        label="Start Time"
-                    >
-                        <Form.Control
-                            type="time"
-                            placeholder="Start Time"
-                            name="startTime"
-                        />
-                    </FloatingLabel>
+                    <h4>Day Start</h4>
+                    <TimePicker
+                        className="timePicker-input"
+                        placement="bottomRight"
+                        placeholder="StartTime"
+                        showSecond={false}
+                        minuteStep={15}
+                        value={startTimeValue}
+                        onChange={setStartTimeValue}
```

Comment:
Since onChange provides the new moment object of the new TimePicker value we can just pass that directly to the setStartTimeValue dispatch function. Essentially we are referencing that dispatch state function rather than having to wrap it in 2 functions like you had it before (handler function and then another function wrapping the handler)

Reply:
```
const onChangeStartTimeHandler = (value) => {
  setStartTimeValue(value);
}
...
<TimePicker
  ...
  onChange={(value) => { onChangeStartTimeHandler(value) }}
/>
```